app/Http/Controllers/Tabbar/Navigation/Docs/DocsController.php

```diff
@@ -3,30 +3,23 @@
 namespace App\Http\Controllers\Tabbar\Navigation\Docs;
 
 use App\Http\Controllers\Controller;
-use App\Settings\PathInfo;
 use Parsedown;
 
-class DocsController extends Controller
-{
-    /*
-    |--------------------------------------------------------------------------
-    | DocsController
-    |--------------------------------------------------------------------------
-    |
-    | Docsコントローラ
-    |
-    */
+/**
+ * DocsController
+ *
+ * Docsタブ内で表示するコンテンツを扱う
+ *
+ */
+class DocsController extends Controller {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition